src/components/dropdowns/dropdown-complex-search/DropdownComplexSearch.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { useState } from 'react';
+import styled from 'styled-components';
+import {
+  ErrorMsgCSS,
+  OptionCSS,
+  OptionsWrapperCSS,
+  WrapperCSS,
+  InputCSS,
+} from './DropdownComplexSearchCSS';
+import { colors } from '../../colors';
+
+import { Search } from '../../../assets/icons';
+
+const Wrapper = styled.div`
+  ${WrapperCSS};
+`;
+
+const Image = styled.img`
+  filter: saturate(0);
+  position: absolute;
+  margin-left: 10px;
+  margin-top: 10px;
+`;
+
+const Input = styled.input`
+  ${InputCSS}
+  padding-left: 40px;
+  border-radius: ${({ showOptions }) => (showOptions ? `7px 7px 0 0 ` : `7px`)};
+  border: ${({ showOptions }) =>
+    showOptions
+      ? `2px solid ${colors.emerald400}`
+      : `2px solid ${colors.grey500}`};
+  color: ${({ selectedOption }) =>
+    selectedOption ? `${colors.emerald400}` : `${colors.grey500}`};
+
+  &:hover {
+	color: ${({ error }) => (error ? `white` : ` ${colors.emerald400}`)}
+    border: 2px solid ${colors.emerald400};
+  }
+
+  border: ${({ error }) => (error ? `2px solid ${colors.red500}` : '')};
+`;
+
+const OptionsWrapper = styled.div`
+  ${OptionsWrapperCSS}
+`;
+
+const Option = styled.div`
+  ${OptionCSS}
+  width: 320px;
+  border-radius: ${({ isLastOption }) => (isLastOption ? '0 0 6px 6px' : '')};
+  margin-bottom: ${({ isLastOption }) => (isLastOption ? '-1px' : '')};
+`;
+
+const ErrorMsg = styled.div`
+  ${ErrorMsgCSS}
+`;
+
+const DropdownComplexSearchComponent = ({ placeholder, options, error }) => {
+  const [showOptions, setShowOptions] = useState(false);
+  const [selectedOption, setSelectedOption] = useState('');
+  const [hasError, setHasError] = useState(error);
+
```

Comment:
I would add a useeffect here so that the error styles don't continue showing after a user types in the input

  ```
useEffect(() => {
    if (hasError && selectedOption !== '') {
      setHasError(false);
    }
  }, [hasError, selectedOption]);
```